app/controllers/update_task_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UpdateTaskController < ApplicationController
+  def completed
+    Task.update(params[:id], :completed_at => Time.new)
+
+    redirect_to "/"
+  end
+
+  def edit
+    @task = Task.find(params[:id])
+  end
+
+  def update
+    task_params = create_params[:task]
+    @task = Task.update(params[:id],
+                        :name => task_params[:name],
```

Comment:
I have seen it used directly before and I'm guessing I could use it here too. Does it only work if the symbol names in the params and the database column names are the same?